core/keysign/tss/models.ts

```diff
@@ -0,0 +1,162 @@
+export namespace storage {
+  export class AddressBookItem {
```

Comment:
we should not create this , as the class is created by wails when build

Reply:
Resolved